content/en/docs/aws/rds.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Configure External Database Using Amazon RDS"
+date = 2020-04-29T12:54:30-04:00
+description = "Using Amazon RDS for storing pipelines and metadata"
+weight = 98
++++
+
+This guide describes how to use Amazon RDS as your pipelines and metadata store.
+
+## Amazon Relational Database Service (Amazon RDS)
+
+Amazon RDS is a managed service that makes it easier to set up, operate, and scale a relational database in the AWS Cloud. It provides cost-efficient, resizable capacity for an industry-standard relational database and manages common database administration tasks. It has support for several engines such as  MySQL, MariaDB, PostgreSQL, Oracle, and Microsoft SQL Server DB engines.  
+
+### Deploy Amazon RDS MySQL in your environment
+
+Before deploying MySQL database using Amazon RDS, let's get configuration parameters that are needed such as VpcId, SubnetIds and SecurityGroupId
+
+```shell
+# Use these commands to find VpcId, SubnetId and SecurityGroupId if you deployed your EKS cluster using eksctl
+# For DIY Kubernetes on AWS, modify tag name or values to get desired results
+export AWS_CLUSTER_NAME=<YOUR EKS CLUSTER NAME>
+
+# Below command will retrieve your VpcId
+aws ec2 describe-vpcs --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$AWS_CLUSTER_NAME | jq -r '.Vpcs[].VpcId'
+
+# Below command will retrieve list of Private subnets
+# You need to use at least two Subnets from your List
+aws ec2 describe-subnets --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$AWS_CLUSTER_NAME Name=tag:aws:cloudformation:logical-id,Values=SubnetPrivate* | jq -r '.Subnets[].SubnetId'
+
+# Below command will retrieve SecurityGroupId for your Worker nodes
+# This assumes all your Worker nodes share same SecurityGroups
+INSTANCE_ID=$(aws ec2 describe-instances --query 'Reservations[*].Instances[*].InstanceId' --filters "Name=tag-key,Values=eks:cluster-name" "Name=tag-value,Values=$AWS_CLUSTER_NAME" --output text)
+for i in "${INSTANCE_IDS[@]}"
+do
+  echo "SecurityGroup for EC2 instance $i ..."
+aws ec2 describe-instances --instance-ids $i | jq -r '.Reservations[].Instances[].SecurityGroups[].GroupId'
+done  
+```
+You can either use console or use attached CloudFormation template to deploy Amazon RDS database
```

Comment:
Minor suggestion: Please add a period(.) at the end of sentence.

Question: so the instruction described below is for console or template?

Reply:
resolved. Instruction is for launching cloudformation template